src/Response/XmlApiResponse.php

```diff
@@ -18,18 +18,25 @@ class XmlApiResponse implements ApplicationIResponse
      */
     private $response = null;
 
+    /**
+     * @var string|int|bool|DateTimeInterface
+     */
+    private $expiration;
+
     /**
      * Create XmlApiResponse
      * This class only wrap JsonResponse from Nette and add possibility
      * to setup response code and automaticaly set content type
      *
      * @param integer $code
      * @param string $data
+     * @param string|int|bool|DateTimeInterface $expiration
      */
-    public function __construct($code, $data)
+    public function __construct($code, $data, $expiration = false)
```

Comment:
I would prefer something consistent :) let's say DateTimeInterface|null and you can use it as typehint here

Reply:
ok php version is 5.6 :) I take it back :)

Reply:
this version si for php 5.6, there is no type hint for null :(